nsg.tf

```diff
@@ -50,14 +50,3 @@ resource "aws_security_group_rule" "nsg_task_egress_rule" {
   security_group_id = aws_security_group.nsg_task.id
 }
 
-resource "aws_security_group_rule" "egress_to_graylog_rule" {
```

Comment:
Can you please remove all the graylog stuff from this PR? @kelemensanyi is currently testing this with a live app.